### PR TITLE
Update aspredicted template

### PR DESCRIPTION
**Pull request by JeffreyRStevens**
This PR updates the skeleton.Rmd text for `aspredicted_prereg` to match the updated form on https://aspredicted.org/create.php as of 2021-01-16.
Changes include:
- Rename Existing data to Data collection
- Add "It's complicated' and update Yes and No entries under Data collection
- Replace More analyses with Outliers and exclusions
- Add Name
- Add Study type
- Standardize capitalization of headings

### CODE REVIEW

File: inst/rmarkdown/templates/aspredicted_prereg/skeleton/skeleton.Rmd [hunk 1]
```diff
@@ -55,22 +58,37 @@ output: prereg::aspredicted_prereg
...
 
 
 ## Other
-<!-- Anything else you would like to pre-register? (e.g., data exclusions, variables collected for exploratory purposes, unusual analyses planned?) -->
+<!-- Anything else you would like to pre-register? (e.g., secondary analyses, variables collected for exploratory purposes, unusual analyses planned?) -->
+
+> Example: We will include a battery of questions for exploratory purposes, including life satisfaction, amount of videogame playing, and family activity. We will also provide an additional survey with 24 questions assessing achievement orientation. We will not report the results of those analyses for the project being pre-registered.
+> NOTE: If you leave this blank it will read 'Nothing else to pre-register.'
+
+## Name
```
Comment (crsh) [inline inst/rmarkdown/templates/aspredicted_prereg/skeleton/skeleton.Rmd]: Given that the YAML header has a title field, do we need this section?

**Review by crsh — COMMENTED** [review]
Excellent, thank you very much! I just added one minor comment. Also, feel free to add yourself to `DESCRIPTION` as a contributor.

**Comment by JeffreyRStevens** [comment]
Good catch on the redundant Name field. I have removed the Name field and added myself as a contributor. Thanks!
Let me know if you have any other recommendations.

**Review by crsh — APPROVED** [review]
:+1:

**Pull request merged**